devdocs/orientation-guide.md

```diff
@@ -0,0 +1,120 @@
+
+Roles:
+
+* **Tester**: Tests the App manually and reports bugs or suggestions for enhancements in the issue tracker.
+*   **Copy editor**: Helps in improving documentation.
+*   **Web page developer**: Works on static web pages, such as those used in the TEAMMATES web site
+*   **Test developer**: Works on automating system tests.
+*   **Frontend developer**: Works on the frontend of the App that is generated dynamically from the server side.
+*   **Backend developer**: Works on the backend logic of the App, including data storage.
+
+## Dev Community Structure
+
+[TEAMMATES community](https://teammatesv4.appspot.com/about.html) has three tiers, defined based on the level of involvement.
+
+*   Contributor: Small but noteworthy contribution to the project.
+*   Committer: Significant contributions sustained over a long period of time.
+*   Core member: Significant and sustained contributions in recent times. Core members can progress through the levels: Snr Developer, Area Lead, Project Lead, and Project Manager.
+
+## Getting started
+
+Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resource when you cannot. To become a TEAMMATES contributor, you need to start honing those skills. To help you with that, we have created a sequence of tasks you can try to complete.  Try to complete as many of them as you can, in the order they are listed. Of course we are happy to guide you if you encounter any difficulties when doing these tasks; just post your question in [our issue tracker](https://github.com/TEAMMATES/teammates/issues). If you are seeking help on project setup, remember to include the following in your post.
+
+1. Step in the setting up guide that you are in
+
+2. What measures you took to address the issue
+
+3. Operating system
+
+4. Screenshots (if any)
+
+
+> Note: The task descriptions are brief by intention. We want you to try to figure out how to do those things by yourself.
+
+## Orientation task list
+
+_**Phase A**: Know the project_
+
+1.  Understand what TEAMMATES is about.
+    * [product intro page](https://teammatesv4.appspot.com) shown to potential users
+
+    * [project vision, challenges, and principles](./overview.md)
+    * [feature overview](https://teammatesv4.appspot.com/features.html) (users’ point of view)
+
+2.  Decide in which role(s) you want to contribute. i.e., tester, copy-editor, etc. as specified [earlier in this document](#knowledge-required).
+3.  If you are aiming for tech writer or tester roles, get an instructor account using the link in the TEAMMATES home page. Remember to mention the purpose of your request under any other comments.
+
+    If you are sure you want to become a contributor, you can skip this step. In Phase B, you get to set up TEAMMATES on your own machine and use that instance to try the functionality.
+
+    If you plan to contribute code, continue with the tasks below.
```

Comment:
Number 3 here is actually part of number 2. Make it like this:

2. Decide ... as specified earlier in this document.
    * If you are aiming for ... any other comments. If you are sure ... try the functionality.
    * If you plan to ... tasks below.